dandiapi/api/tests/test_upload.py

```diff
@@ -38,7 +38,7 @@ def test_blob_read_bad_algorithm(api_client, asset_blob):
         format='json',
     )
     assert resp.status_code == 400
-    assert resp.data == 'Unsupported Digest Algorithm'
+    assert resp.data.startswith('Unsupported Digest Algorithm. Supported:')
```

Comment:
```suggestion
    assert resp.data == 'Unsupported Digest Algorithm. Supported: dandi:dandi-etag'
```
We should be forced to update this test if we ever add new supported digest types.